Makefile

```diff
@@ -313,6 +313,27 @@ test.integration.enterprise.postgres.pretty:
 		COVERAGE_OUT=coverage.enterprisepostgres.out 2>/dev/null | \
 		$(GOTESTFMT_CMD)
 
+.PHONY: test.integration.cp
+test.integration.cp:
```

Comment:
That target is not used anywhere. Did you meant `_test.integration.cp`?

Reply:
Yes, nice caught. Thanks 😄